src/components/Navbar.js

```diff
@@ -74,6 +74,7 @@ class Navbar extends React.PureComponent {
                     <Slider
                         min={0}
                         max={1}
+                        step={0.01}
```

Comment:
This seemed to be needed, otherwise it acted like `step=1`. I suspect I might have a newer version of material-ui installed.